web-services/query/src/main/java/datawave/webservice/query/runner/RunningQuery.java

```diff
@@ -110,7 +110,10 @@ public RunningQuery(QueryMetricsBean queryMetrics, Connector connection, Accumul
         this.connectionPriority = priority;
         this.settings = settings;
         this.calculatedAuths = AuthorizationsUtil.getDowngradedAuthorizations(methodAuths, principal);
-        this.timing = timing;
+        this.timing = (RunningQueryTimingImpl) timing;
+        if (this.timing == null) {
```

Comment:
We check for timing being null below... no need for this

Reply:
Removed